.gitignore

```diff
@@ -31,4 +31,7 @@ yarn-error.log*
 .env.production.local
 
 # vercel
-.vercel
+.
+
+# package-json.lock
+package-json.lock
```

Comment:
Let's not ignore it

Reply:
Removed package-json.lock from .gitignore